core/packet.cc

```diff
@@ -15,6 +15,8 @@
 
 namespace bess {
 
+const size_t PacketBatch::kMaxBurst;
```

Comment:
It is recommended to also initialize the variable in the source file, rather than in the class.